src/js/formapp.js

```diff
@@ -208,7 +210,12 @@ const Router = Backbone.Router.extend({
 });
 
 function startFormApp() {
-  $('#root').append(`<div class="spinner"><div class="spinner-circle">${ '<div class="spinner-child"></div>'.repeat(12) }</div></div>`);
+  $('#root').append(`
+    <div class="preloader__bar js-progress-bar">
+      <div class="preloader__bar-progress"></div>
+    </div>
+    <div class="preloader__text js-loading">${ intl.regions.preload.loading }</div>
+  `);
```

Comment:
This is for the loader that is shown in the form iframe.

Before this PR, the spinner was shown without the `Loading...` text.

Do we want the `Loading...` text here? Or just the progress bar animation by itself?